PyInstaller/hooks/hook-botocore.py

```diff
@@ -16,8 +16,14 @@
 #
 # https://botocore.readthedocs.org/en/latest/
 #
-# Tested with botocore 1.3.0
+# Tested with botocore 1.4.36
```

Comment:
#2069 already includes this code, but requires some code-styling. So I'm closing this pull-request as being duplicate.